Source/Core/Core/Boot/Boot_WiiWAD.cpp

```diff
@@ -29,7 +28,7 @@ bool CBoot::BootNANDTitle(const u64 title_id)
   if (ticket.IsValid())
     console_type = ticket.GetConsoleType();
   else
-    ERROR_LOG(BOOT, "No ticket was found for %016" PRIx64, title_id);
+    ERROR_LOG_FMT(BOOT, "No ticket was found for {:016X}", title_id);
```

Comment:
nitpick (we mostly use lowercase for title IDs):

```suggestion
    ERROR_LOG_FMT(BOOT, "No ticket was found for {:016x}", title_id);
```

Reply:
Done